samples/dfu/README.rst

```diff
@@ -211,14 +237,14 @@ Run following command on host PC to upload new firmware as artifact to Golioth:
 
 .. code-block:: console
 
-   $ goliothctl artifact create new.bin --version 1.2.3
+   $ goliothctl dfu artifact create new.bin --version 1.2.3
 
 Then create new release consisting of this single firmware and roll it out to
 all devices in a project:
 
 .. code-block:: console
 
-   $ goliothctl release --release-tags 1.2.3 --components main@1.2.3 --rollout true
+   $ goliothctl dfu release create --release-tags 1.2.3 --components main@1.2.3 --rollout true
```

Comment:
Those lines should be fixed in separate commit. Bugfixes separately, new features separately.